lib/pages/chat/chat_view.dart

```diff
@@ -29,8 +29,9 @@ class ChatView extends StatelessWidget with MessageContentMixin {
           TwakeIconButton(
             icon: Icons.copy_outlined,
             tooltip: L10n.of(context)!.copy,
-            onTap: () => controller
-                .actionWithClearSelections(controller.copySingleEventAction),
+            onTap: () => controller.actionWithClearSelections(
+              () => controller.copySingleEventAction(),
+            ),
```

Comment:
what is the different?

Reply:
Forgot to remove it after debugging my bad